source/frontend/src/components/maps/MapSearch.tsx

```diff
@@ -67,7 +67,6 @@ const StyledFilterContainer = styled(Container)`
       border-width: 0.2rem;
     }
     .btn-primary {
-      color: white;
```

Comment:
Is this intentional?

Reply:
yes - this would interfere with the "disabled" state of the map search button (second screenshot I showed)